modules/mono/build_scripts/build_assemblies.py

```diff
@@ -277,6 +281,8 @@ def build_all(msbuild_tool, module_dir, output_dir, godot_platform, dev_debug, p
     args = ["/restore", "/t:Build", "/p:Configuration=Release"]
     if push_nupkgs_local:
         args += ["/p:ClearNuGetLocalCache=true", "/p:PushNuGetToLocalSource=" + push_nupkgs_local]
+    if float_size == 64:
```

Comment:
```suggestion
    if float_size == "64":
```
...since the parameter `--float` is now typed as`str`